test/es6/rlexe.xml

```diff
@@ -1059,6 +1059,13 @@
       <tags>BugFix</tags>
     </default>
 </test>
+<test>
+    <default>
+      <files>ES6NewTarget_bugfixes.js</files>
+      <compile-flags>-ES6Classes -forceundodefer -args summary -endargs</compile-flags>
```

Comment:
The function that has `new.target` is defined in the global function (`tests[2]["body"]`).
Without the change in `parse.cpp`, this test fails with `-forceundodefer`